web/src/pages/apd/activities/contractor-costs/ContractorResource/ContractorResourceForm.js

```diff
@@ -90,6 +90,7 @@ const ContractorResourceForm = forwardRef(
           disabled
           style={{ display: 'none' }}
           aria-hidden="true"
+          aria-label="submitButton"
```

Comment:
Are these labels read to screen readers?

Reply:
Just tested. No they aren't read out loud